rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -80,6 +93,13 @@ pub const SECOND: Duration = Duration::from_secs(1);
 const KB: usize = 1024;
 const MB: usize = 1024 * KB;
 
+// Log Anonymization
+
+lazy_static! {
+    static ref LOG_ANONYMIZATION_CID: Principal =
```

Comment:
Move that from globals to CLI?

Reply:
Done, now it's passed as a CLI argument 👍